supertokens_python/recipe/session/recipe_implementation.py

```diff
@@ -338,6 +418,92 @@ async def get_access_token_lifetime_ms(self, user_context: Dict[str, Any]) -> in
     async def get_refresh_token_lifetime_ms(self, user_context: Dict[str, Any]) -> int:
         return (await self.get_handshake_info()).refresh_token_validity
 
+    async def merge_into_access_token_payload(
+        self,
+        session_handle: str,
+        access_token_payload_update: Dict[str, Any],
+        user_context: Dict[str, Any],
+    ) -> bool:
+        session_info = await self.get_session_information(session_handle, user_context)
+        if session_info is None:
+            return False
+
+        new_access_token_payload = {
+            **session_info.access_token_payload,
+            **access_token_payload_update,
+        }
+        for k in access_token_payload_update.keys():
+            if new_access_token_payload[k] is None:
+                del new_access_token_payload[k]
```

Comment:
if `k` does not exist, would `if new_access_token_payload[k] is None:` be true?

Reply:
It's impossible that `k` won't exist in `new_access_token_payload` because 
```python
new_access_token_payload = {
      **session_info.access_token_payload,
     **access_token_payload_update,
}
```

but even assuming that this magically happens, python will throw error if `k` doesn't exist. So the condition can't be true.